{{cookiecutter.project_slug}}/clients/vue3/src/views/PageNotFound.vue

```diff
@@ -1,11 +1,9 @@
 <template>
-  <div class="page-not-found">
-    <h1 class="page-not-found__error">404</h1>
+  <div class="flex flex-col items-center">
+    <h1 class="text-7xl font-extrabold">404</h1>
     <img alt="Project Logo" src="../assets/logo.png" />
-    <h2 class="page-not-found__header">Oops!<br />We couldn't find the page you're looking for.</h2>
-    <p class="page-not-found__route">
-      Click <router-link to="/">here</router-link> to return to the home page.
-    </p>
+    <h2 class="text-xl font-bold mb-4">We couldn't find the page you're looking for.</h2>
```

Comment:
**Info:** Removed the "oops" text, as the tone is not necessarily appropriate. With error handling, we want to show the users that we know it’s serious and we understand it’s important to them (and not necessarily be more "cutesy" here).

Reference article: [Wix-UX: When life gives you lemons, write better error messages](https://wix-ux.com/when-life-gives-you-lemons-write-better-error-messages-46c5223e1a2f)